### PR TITLE
Add a profile page

### PR DESCRIPTION
**Pull request by jyx11011**
profile page with hard-coded data
Screenshot 2020-06-26 at 2 28 55 PM
editing profile
Screenshot 2020-06-26 at 2 29 17 PM

### CODE REVIEW

**Comment by jyx11011** [comment]
@googlebot I signed it!

**Comment by jyx11011** [comment]
@googlebot I signed it!

File: src/main/webapp/js/util.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+	var oldonload = window.onload;
+	if(typeof oldonload == "function") {
+		window.onload = function() {
+			oldonload();
+			func();
+		}
+	} else {
+		window.onload = func;
+    }
+}
```
Comment (limli) [inline src/main/webapp/js/util.js:16]: run checkstyle on this file. Change tabs to spaces, function should have jsdocs.

File: src/main/webapp/js/profile.js [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+{
+  'id': 1234,
+  'name': 'Aaron Tan',
+  'username': 'aarontan',
+  'email': 'aaront@example.com',
+  'picture': 'images/profile_pic.svg',
+  'bio': 'a short bio of aaron tan',
+  'dealsUploaded': [
+    {
+        'id': 1234,
```
Comment (limli) [inline src/main/webapp/js/profile.js]: run eslint on this file. Number of spaces are wrong.

File: src/main/webapp/js/profile.js [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+    },
+    {
+      'id': 23452,
+      'name': 'sushi'
+    }
+  ]
+}
+
+/** Configures user's profile. */
+function configureUserProfile(user) {
```
Comment (limli) [inline src/main/webapp/js/profile.js]: jsdocs should have parameter and @return

**Comment by limli** [comment]
looks good to me

**Review by SuyashLakhotia — DISMISSED** [review]

**Review by SuyashLakhotia — APPROVED** [review]

**Pull request merged**